src/Support/Support/ChangeLog.md

```diff
@@ -20,6 +20,17 @@
 
 ## Upcoming Release
 
+## Version 1.0.0
```

Comment:
remove this line

Reply:
Removed

Reply:
Also reverted the change to AssemblyInfo.cs, Az.Support.psd1